drsyscall/drsyscall.c

```diff
@@ -247,13 +250,21 @@ drsys_name_to_syscall(const char *name, drsys_syscall_t **syscall OUT)
         ASSERT(false, "name2num should return num in systable");
         return DRMF_ERROR_NOT_FOUND;
     }
+
 #ifdef DEBUG
-    ASSERT(stri_eq(sysinfo->name, name)
+    /* ignore possible Nt/Zw mismatch */
+    if (((sysinfo->name[0] == 'N' && sysinfo->name[1] == 't') ||
```

Comment:
Why the next commit?  It needs to update to master and thus re-run the checks anyway, right?